frontend/src/main/scala/bloop/Bloop.scala

```diff
@@ -53,6 +53,16 @@ object Bloop {
         val action = Run(command, Exit(ExitStatus.Ok))
         run(Interpreter.execute(action, state))
 
+      case Array("runMain", projectName, mainClass, args @ _*) =>
```

Comment:
If the shell is becoming a `CaseApp`, isn't this unnecessary? You may need to adapt it in your other PR, no need to block this one.

Reply:
It's a `CaseApp`, but you still use the shell, so it doesn't affect what you type in. Or am I misunderstanding your comment?

Reply:
Solved in a private discussion